doc/python/choropleth-maps.md

```diff
@@ -33,7 +33,7 @@ jupyter:
     thumbnail: thumbnail/choropleth.jpg
 ---
 
-A [Choropleth Map](https://en.wikipedia.org/wiki/Choropleth_map) is a map composed of colored polygons. It is used to represent spatial variations of a quantity. This page documents how to build **outline** choropleth maps, but you can also build [choropleth **tile maps** using our Mapbox trace types](/python/mapbox-county-choropleth).
+A [Choropleth Map](https://en.wikipedia.org/wiki/Choropleth_map) is a map composed of colored polygons. It is used to represent spatial variations of a quantity. This page documents how to build **outline** choropleth maps, but you can also build [choropleth **tile maps**](/python/tile-county-choropleth).
```

Comment:
Maybe we should have in all of these charts a banner like:
> These trace types replace the previous `_mapbox` trace types, which will be deprecated in a feature version of Plotly. The core difference is that these traces use Maplibre as the rendering service rather than Mapbox, which allows us to keep our chart types up to the latest standards.

Or something? 

IMHO with this text we might also post on the forums and say, "If you have any feedback or want to learn more, you can see this post on our forums."

cc @gvwilson ; what do you think?

Reply:
I like the first sentence ("These trace types..."), but rather than details about Maplibre vs Mapbox, put in a URL to a page like the one we did for maintenance badges so that we can update as questions come in or our decisions change w/out having to do another release?